src/huggingface_hub/_upload_large_folder.py

```diff
@@ -44,6 +45,7 @@
 WAITING_TIME_IF_NO_TASKS = 10  # seconds
 MAX_NB_FILES_FETCH_UPLOAD_MODE = 100
 COMMIT_SIZE_SCALE: List[int] = [20, 50, 75, 100, 125, 200, 250, 400, 600, 1000]
+TARGET_NB_FILES_XET_UPLOAD_BATCH_SIZE = 256  # target nb files per upload batch for XET-enabled repos
```

Comment:
```suggestion

UPLOAD_BATCH_SIZE_XET = 256  # Max 256 files per upload batch for XET-enabled repos
UPLOAD_BATCH_SIZE_LFS = 1  # Otherwise, batches of 1 for regular LFS upload
```
I will define the `1` for LFS upload as a constant as well